src/index.ts

```diff
@@ -7,75 +7,83 @@ import debug from './debug'
 import config from './config'
 import jwks from 'jwks-rsa'
 import jwt from 'express-jwt'
+import {getTitle} from "./translations";
 
 const protect = jwt({
-    // Dynamically provide a signing key based on the kid in the header and the signing keys provided by the JWKS endpoint.
-    secret: jwks.expressJwtSecret({
-        cache: true,
-        jwksRequestsPerMinute: 5,
-        jwksUri: config.jwksUrl,
-    }),
-    algorithms: ['RS256'],
+  // Dynamically provide a signing key based on the kid in the header and the signing keys provided by the JWKS endpoint.
+  secret: jwks.expressJwtSecret({
+    cache: true,
+    jwksRequestsPerMinute: 5,
+    jwksUri: config.jwksUrl,
+  }),
+  algorithms: ['RS256'],
 })
 
 const app = express()
 app.set('view engine', 'hbs')
 
 app.use((req: Request, res: Response, next: NextFunction) => {
-    res.locals.projectName = config.projectName;
-    res.locals.baseUrl = config.baseUrl;
-    next();
+  res.locals.projectName = config.projectName;
+  res.locals.baseUrl = config.baseUrl;
+  next();
 })
 app.use(express.static('public'))
 app.use(express.static('node_modules/normalize.css'))
 
 app.engine(
-    'hbs',
-    handlebars({
-        extname: 'hbs',
-        layoutsDir: `${__dirname}/../views/layouts/`,
-        partialsDir: `${__dirname}/../views/partials/`,
-        defaultLayout: 'main',
-        helpers: {
-            json: (context: any) => JSON.stringify(context),
-            jsonPretty: (context: any) => JSON.stringify(context, null, 2),
-        }
-    })
+  'hbs',
+  handlebars({
+    extname: 'hbs',
+    layoutsDir: `${__dirname}/../views/layouts/`,
+    partialsDir: `${__dirname}/../views/partials/`,
+    defaultLayout: 'main',
+    helpers: {
+      json: (context: any) => JSON.stringify(context),
+      jsonPretty: (context: any) => JSON.stringify(context, null, 2),
+      getTitle,
+      whichFormInput: (type: string) =>
```

Comment:
Add a small comment to explain why we need this. I also think that a switch statement would be more readable. Remember that we're writing this as a reference for other developers - the more readable it is, the better!

Reply:
I'd also suggest renaming this to `toFormFieldTemplateName` or something along those lines